src/main/java/org/ipvp/canvas/paginate/PaginatedMenuBuilder.java

```diff
@@ -134,8 +135,9 @@ public PaginatedMenuBuilder addSlotSettings(Collection<SlotSettings> slotSetting
 
         do {
             Menu page = getPageBuilder().build();
-            if (getNewMenuModifier() != null) {
-                getNewMenuModifier().accept(page);
+            if (!getNewMenuModifiers().isEmpty()) {
+                for (Consumer<Menu> menuModifier : getNewMenuModifiers())
```

Comment:
Please add braces to if-statements
```suggestion
                for (Consumer<Menu> menuModifier : getNewMenuModifiers()) {
```